packages/runtime/container-runtime/src/gc/gcTelemetry.ts

```diff
@@ -288,17 +284,14 @@ export class GCTelemetryTracker {
 			details: { ...detailedProps, ...additionalProps }, // Also includes some properties from INodeUsageProps type
 			gcConfigs,
 			tombstoneFlags: {
-				DisableTombstone: this.mc.config.getBoolean(disableTombstoneKey),
-				ThrowOnTombstoneUsage: this.mc.config.getBoolean(throwOnTombstoneUsageKey),
 				ThrowOnTombstoneLoad: this.mc.config.getBoolean(throwOnTombstoneLoadOverrideKey),
```

Comment:
I think you can remove this too, it can be inferred from gcConfigs (and I'm suggesting you remove this config anyway, since we have to have the runtimeOption for AzureClient compat mode "1")